### PR TITLE
Automated rsyslog clusterlogforwarder test cases: #OCP-33627,32643,32967,32864

### PR DESCRIPTION
**Pull request by gkarager**
@anpingli @QiaolingTang please do the review.

### CODE REVIEW

File: features/logging/cluster_log_forwarder.feature [hunk 1]
```diff
@@ -295,3 +295,83 @@ Feature: cluster log forwarder features
...
+    And I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj clipboard
+    Given I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-logging" project
+
+    Given I obtain test data file "logging/clusterlogging/example_indexmanagement.yaml"
```
Comment (anpingli) [inline features/logging/cluster_log_forwarder.feature:330]: Line 330-349, I  don't think we have to test the ES here. 
 How about use the fluent-only template?  I
    Given I obtain test data file "logging/clusterlogging/fluentd_only.yaml"
    When I create clusterlogging instance with:
      | remove_logging_pods | true              |
      | crd_yaml            | fluentd_only.yaml |
      | check_status        | false

Comment (anpingli) [reply features/logging/cluster_log_forwarder.feature:330]: And I think if we create clusterlogging/instance after  clusterlogforwarder/instance. The script will be more simple.

**Comment by gkarager** [comment]
I have run the cases with runner-v3. Here is the pass log link: https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/212826/console

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**